File: day9/count_vowels.py
vowels = ['a', 'e', 'i', 'o', 'u']

user_input = input("Enter a word or words and I will count the number of vowels: ")
vowel_count = sum([1 for i in user_input.lower() if i in vowels])
print(f"The number of vowels in your input is {vowel_count} (not including y).")

File: day9/CountVowels.java
import java.util.Scanner;

public class CountVowels {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a word or words and I will count the number of vowels: ");
        String input = scanner.nextLine().toLowerCase();

        int vowelCount = 0;
        for (char c : input.toCharArray()) {
            if ("aeiou".indexOf(c) != -1) {
                vowelCount++;
            }
        }

        System.out.println("The number of vowels in your input is " + vowelCount + " (not including y).");
        scanner.close();
    }
}
